playground/AzureFunctionsEndToEnd/AzureFunctionsEndToEnd.ApiService/Program.cs

```diff
@@ -0,0 +1,55 @@
+using System.Text;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Producer;
+using Azure.Storage.Blobs;
+using Azure.Storage.Queues;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add service defaults & Aspire components.
+builder.AddServiceDefaults();
+builder.AddAzureQueueClient("queue");
+builder.AddAzureBlobClient("blob");
+builder.AddAzureEventHubProducerClient("eventhubs", static settings => settings.EventHubName = "myhub");
+
+var app = builder.Build();
+
+app.MapGet("/publish/asq", async (QueueServiceClient client, CancellationToken cancellationToken) =>
```

Comment:
What do you think about simplifying this example, and just putting these as buttons on the web front end?

That way someone only needs to reason about the web app, and the functions app.

For example, right now in the AppHost Program.cs, the webfrontend has a reference to apiService, but it doesn't appear to use/call it.

Reply:
> I think if we wanted the sample to be realistic

These "playground apps" aren't necessarily realistic and aren't intended to be samples. They are just intended for us to be able to try things out ourselves. Keeping them simple is a virtue so we don't have to maintain more than we need.

Reply:
We ended up simplifying it the other way around by removing the web UI from this sample app altogether. I think it's important to keep the API -> queue -> Azure Functions flow so this feels like a good landing place.